region/lowernorfair/west/Acid Statue Room.json

```diff
@@ -289,6 +289,7 @@
         ]},
         {"heatFrames": 285}
       ],
+      "flashSuitChecked": true,
```

Comment:
The note suggests `canPartialFloorClip` is here for the tech component, and you actually do a spinjump